src/uct/ib/mlx5/dv/ib_mlx5_dv.c

```diff
@@ -116,6 +116,7 @@ ucs_status_t uct_ib_mlx5_devx_create_qp(uct_ib_iface_t *iface,
     UCT_IB_MLX5DV_SET(qpc, qpc, pm_state, UCT_IB_MLX5_QPC_PM_STATE_MIGRATED);
     UCT_IB_MLX5DV_SET(qpc, qpc, pd, dvpd.pdn);
     UCT_IB_MLX5DV_SET(qpc, qpc, uar_page, uar->uar->page_id);
+    ucs_assert(attr->srq_num != 0);
```

Comment:
then why in next line we check !!attr->srq_num?

Reply:
it always has to be `0x1` then

Reply:
AFAIR preparation for Devx UD

Reply:
> AFAIR preparation for Devx UD

thank you!
fixed `ucs_assert`